### DISCUSSION TITLE
is it possible to return a checkbox group as array of values, not objects?

### DISCUSSION THREAD

**Discussion by danielzzz** [react-hook-form/react-hook-form#4077] [General]
hi, I am trying to get selected checkbox group values from the form.

I've seen some possible solutions for that, but all of them return all values as object where some property determines if it has been checked or not, like in this example:
https://codesandbox.io/s/funny-mayer-8ojxq

```
[{selected: "1"}, {selected: false}]
```
I was wondering if it is possible to configure it in a way that it would return just an array of actually selected values:
```
[1]
```

**Comment by bluebill1049** [comment]
you will have to build a controlled component and wrapped with Controller or use the custom hook version of useController.
